index.html

```diff
@@ -6,11 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>API Practice</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <div class="container"></div>
-    <button id="random">Get Info</button>
+    <div class="container">
+        <p id="randomTitle"></p>
```

Comment:
you can make JS create a new card automatically without needing to write them manually .